client/actions/checks.js

```diff
@@ -4,34 +4,35 @@ import { PENDING, SUCCESS, ERROR } from '../actions/status'
 export const PUT_CHECK = Symbol('put check')
 export const DELETE_CHECK = Symbol('delete check')
 
-const putCheck = (status, payload = null) => ({
+const putCheck = (status, check, payload = null) => ({
   type: PUT_CHECK,
   status,
+  check,
```

Comment:
Agree, I think he situation was that I didn't have an `action` in case of error and this was the quick fix. Will check again.